msi/release.sh

```diff
@@ -44,52 +141,81 @@ SCRIPT_WINPATH="$(cd "$SCRIPT_PATH" && pwd -W | tr / \\\\)"
 BUILD_EXTRA_WINPATH="$(cd "$SCRIPT_PATH"/.. && pwd -W | tr / \\\\)"
 (cat <<EOF
 <?xml version="1.0" encoding="utf-8"?>
+<?define SixtyFourBit$(echo =${BITNESS#32}) ?>
```

Comment:
This is not right. On 32-bit, this will result in `<?define SixtyFourBit= ?>` when it should only be `<?define SixtyFourBit ?>`.